pkg/extension/workspace/config/helpers.go

```diff
@@ -15,12 +15,14 @@
 package config
 
 import (
-	"github.com/ghodss/yaml"
+	"sigs.k8s.io/yaml"
 )
 
 var (
 	// Marshal converts an object into its YAML representation.
-	Marshal func(interface{}) ([]byte, error) = yaml.Marshal
+	Marshal = yaml.Marshal
 	// Unmarshal converts YAML into an object.
-	Unmarshal func([]byte, interface{}) error = yaml.Unmarshal
+	Unmarshal = func(bytes []byte, i interface{}) error {
+		return yaml.Unmarshal(bytes, i) // no opts
+	}
```

Comment:
should be simplified..
```suggestion
	Unmarshal = yaml.Unmarshal
```

Reply:
sorry this was something from the last review also. the comment // no opts is there because we can't shorten it because yaml.Unmarshal has `, opts ...JSONOpt` param...

somehow I lost this change in merging the prior which is why you are seeing it again. sorry about that

Reply:
well `make build` and `make test` pass even if just aliasing `Unmarshal = yaml.Unmarshal`.. so just curious what do you mean by "we can't"..?

Reply:
so it has to do with what the point is of this wrapper.

if you alias like this, it won't compile: `Unmarshal = func(bytes []byte, i interface{}) yaml.Unmarshal`

if you alias without bounding out the options, it will, but then what is the point as you've now leaked the type of the option. If that's the case, why not use yaml.Unmarshal directly

Reply:
iotw I think if we are blindly aliasing, it is fine by me but we should just delete helpers.go in that case.. make sense?

Reply:
I'll go with deleting helpers.go as that will also help narrow the dependency on sigs.k8s.io/yaml at the same time

I don't think helpers.go deserves the attention it has gotten :D

Reply:
> if you alias without bounding out the options,

I see. Now I understand. I would just prefer deleting aliasing here..

Reply:
too few place where these aliases are used.. I think it's just complicating the things.
```
$ git grep -E "config.Unmarshal" | grep -v test
pkg/extension/workspace/model/example.go:       err := config.Unmarshal(file.Content, descriptor)
pkg/extension/workspace/model/workspace.go:     err = config.Unmarshal(data, descriptor)
pkg/extension/workspace/toolchain/builtin/factory.go:                   if err := config.Unmarshal(args.Toolchain.Config.Content, cfg); err != nil {
pkg/extension/workspace/toolchain/lookup.go:    err = config.Unmarshal(file.Content, meta)
mathetake@mathetake ~/getenvoy remotes/origin/envoy1.15-istio1.7*
$ git grep -E "config.Marshal" | grep -v test
// nothing!
```

Reply:
OK! and now only one file depends on sigs.k8s.io/yaml